packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -164,20 +165,25 @@ export const PaymentEventType = {
 }
 export type PaymentEventType = PaymentDepositType | PaymentWithdrawType
 
+export interface OutgoingPaymentJSON {
+  id: string
+  accountId: string
+  createdAt: string
+  state: OutgoingPaymentState
+  receivingPayment: string
+  sendAmount: AmountJSON
+  receiveAmount: AmountJSON
+  description?: string
+  externalRef?: string
+  failed: boolean
+  updatedAt: string
+  sentAmount: AmountJSON
+}
+
 export type PaymentData = {
-  payment: {
-    id: string
-    accountId: string
-    createdAt: string
-    state: OutgoingPaymentState
+  payment: Omit<OutgoingPaymentJSON, 'sentAmount' | 'failed'> & {
     error?: string
     stateAttempts: number
-    receivingAccount?: string
-    receivingPayment?: string
-    sendAmount?: AmountJSON
-    receiveAmount?: AmountJSON
-    description?: string
-    externalRef?: string
     outcome: {
```

Comment:
Why do we still need to include this?

Reply:
I think that is a good idea.

Reply:
I opened an issue:
- https://github.com/interledger/rafiki/issues/345

@matdehaast recently mentioned changing the webhook event id to that of the resource. Maybe those could be tackled to together.